frontend/src/slices/auth/auth.slice.ts

```diff
@@ -25,15 +33,34 @@ const { reducer, actions, name } = createSlice({
     builder.addCase(usersActions.updateUser.fulfilled, (state, action) => {
       state.user = action.payload;
     });
-    builder.addMatcher(isAnyOf(signUp.pending, signIn.pending), (state) => {
-      state.dataStatus = DataStatus.PENDING;
+    builder.addCase(emailResetPasswordLink.fulfilled, (state) => {
+      state.dataStatus = DataStatus.FULFILLED;
+      notification.success(
+        'Email with reset password link was send to your email address',
+      );
```

Comment:
I think it shouldn't be here, it should be put into action.

Reply:
I don't think so.  Notification should be send only if action is fulfilled. If you put it into action it will fire every time.

Reply:
I think you should try it!

```ts
  await authApi.sendEmailResetPasswordLink(payload);

  notification.success(
    'Email with reset password link was send to your email address',
  );
```

Reply:
Agreed, it should be in action, after successful request